app/common/lib/ledgerUtil.js

```diff
@@ -221,6 +223,26 @@ const getMediaId = (data, type) => {
         id = data.docid
         break
       }
+    case ledgerMediaProviders.TWITCH:
+      {
+        if (
+          (
+            data.event === 'minute-watched' ||
```

Comment:
these events would be great to put into a module (so it could work like an enum); ex: `twitchEvents.MINUTE_WATCHED`

Reply:
another way you could do this:
if (['minute-watched', 'video-play', 'player_click_playpause', 'vod_seek'].includes(data.event))

Reply:
added new syntax